templates/AureliaSpa/ClientApp/app/components/app/app.ts

```diff
@@ -0,0 +1,19 @@
+import {Aurelia} from 'aurelia-framework';
```

Comment:
Some few formatting issues in this file:
 * missing spaces around `{ ImportedModule }`
 * The `config.map` object looks pretty strange. Not sure if you were trying to line things up with tabs or something, but it's not lined up now. Also some of the key-value pairs have spaces after the colon and others don't. Could you lay this code out more conventionally?